relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -367,6 +371,13 @@ mod tests {
         "GET *"
     );
 
+    span_description_test!(
+        span_description_scrub_data_images,
+        "GET data:image/png;base64,drtfghaksjfdhaeh/blah/blah/blah",
+        "http.client",
+        "GET data:image/*"
```

Comment:
We can relax the scrubbing to include `png` or `base64`, but I don't think it provides any value and complicates a bit the logic.